src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -107,16 +108,18 @@ const SearchResult = ({ form, formMetaInfo }) => {
         title={title}
         recordGAEvent={recordGAEvent}
       />
-
       <dd className="vads-u-margin-y--1 vads-u-margin-y--1 vsa-from-last-updated">
         <dfn className="vads-u-font-weight--bold">Form last updated:</dfn>{' '}
         {lastRevision}
       </dd>
 
-      {benefitCategories && benefitCategories.length > 0 ? (
+      {(benefitCategories && benefitCategories.length > 0) ||
+      vaFormAdministration ? (
         <dd className="vads-u-margin-y--1 vads-u-margin-y--1">
           <dfn className="vads-u-font-weight--bold">Related to:</dfn>{' '}
-          {benefitCategories.map(f => f.name).join(', ')}
+          {benefitCategories.length > 0
+            ? benefitCategories.map(f => f.name).join(', ')
+            : vaFormAdministration}
```

Comment:
There is a little more to this field in the layout file that we may want to check out in case we want the same logic here, https://github.com/department-of-veterans-affairs/vets-website/blob/6ab97cac750e170bee56b1187559cef8d4b8c735/src/site/layouts/va_form.drupal.liquid#L28. Specifically, the part about non-VA forms and employment.

Now that the logic is a little more involved I think I would move this out of the JSX block -

```jsx
let relatedTo = vaFormAdministration;

if (benefitCategories?.length > 0) {
  relatedTo = benefitCategories.map(f => f.name).join(', ');
} 

// ...
{relatedTo && (
    // render
)}
```

Good opportunity for a little unit test too if possible

Reply:
This is great. Thank you @ncksllvn . 
The PR has been updated.